pkgs/os-specific/linux/open-isns/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, openssl, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "open-isns-${version}";
+  version = "0.95";
+
+  src = fetchFromGitHub {
+    owner = "gonzoleeman";
+    repo = "open-isns";
+    rev = "v${version}";
+    sha256 = "1c2x3yf9806gbjsw4xi805rfhyxk353a3whqvpccz8dwas6jajwh";
+  };
+
+  propagatedBuildInputs = [ openssl ];
+  outputs = ["out" "lib" ];
+  outputInclude = "lib";
+
+  installFlags = "etcdir=$out/etc vardir=$out/var/lib/isns";
```

Comment:
Also, it occurred to me that if isns expects to write to vardir, this will fail at runtime. Perhaps vardir needs to be a parameter, if it indeed needs to be set at compile time.